src/hyperion/parameters/schemas/detector_parameters_schema.json

```diff
@@ -21,5 +18,5 @@
             "type": "string"
         }
     },
-    "minProperties": 6
+    "minProperties": 5
```

Comment:
Should: If `expected_energy_ev` is a valid thing to set it should be in the list above. Then if it's optional you should ennumerate the required ones here, have a look round some of the other schemas, they do this.